ws/db/migrations/env.py

```diff
@@ -25,9 +25,8 @@
 import ws.config
 def get_url():
     ws_config_path = config.get_main_option("ws_config_path")
-    f = open(os.path.expanduser(ws_config_path), "r")
-    parser = ws.config.ConfigFileParser("site", "alembic")
-    conf = parser.parse(f, [])
+    parser = ws.config.ConfigParser(ws_config_path)
+    conf = parser.fetch_section('alembic', to_list=False)
```

Comment:
Please always use double quotes for strings.

Reply:
Ok. :)